lib/mssql.js

```diff
@@ -401,28 +401,24 @@ MsSQL.prototype.applyPagination =
     }
   };
 
-MsSQL.prototype.buildExpression =
-  function(columnName, operator, columnValue, propertyValue) {
-    switch (operator) {
-      case 'like':
-        return new ParameterizedSQL({
-          sql: columnName + " LIKE ? ESCAPE '\\'",
-          params: [columnValue]
-        });
-      case 'nlike':
-        return new ParameterizedSQL({
-          sql: columnName + " NOT LIKE ? ESCAPE '\\'",
-          params: [columnValue]
-        });
-      case 'max':
-      //
-      default:
-        // Invoke the base implementation of `buildExpression`
-        var exp = this.invokeSuper('buildExpression',
-          columnName, operator, columnValue, propertyValue);
-        return exp;
-    }
-  };
+MsSQL.prototype.buildExpression = function(columnName, operator, operatorValue,
+    propertyDefinition) {
+  switch (operator) {
+    case 'like':
+      return new ParameterizedSQL(columnName + " LIKE ? ESCAPE '\\'",
+          [columnValue]);
+    case 'nlike':
+      return new ParameterizedSQL(columnName + " NOT LIKE ? ESCAPE '\\'",
+          [columnValue]);
+    case 'regexp':
+      console.warn('Microsoft SQL Server doe not support the regular ' +
+          'expression operator');
```

Comment:
@raymondfeng I'm suggesting we print a warning and let the operation continue and error out in the default block. This is consistent with the behaviour of the other connectors since we decided to not go with the try/catch route. Thoughts?